tools/@aws-cdk/ubergen/bin/ubergen.ts

```diff
@@ -554,7 +572,6 @@ const IGNORED_FILE_NAMES = new Set([
   '.npmignore',
   'node_modules',
   'package.json',
-  'test',
```

Comment:
Should this really be removed? Looks like it should stay, based on a quick skim of where it's used.

Reply:
Yep